mlir/include/mlir/Dialect/Tensor/TransformOps/TensorTransformOps.td

```diff
@@ -169,4 +169,17 @@ def MakeLoopIndependentOp
   }];
 }
 
+def TypeConversionCastOp : Op<Transform_Dialect,
```

Comment:
This could use a name that is more descriptive of what is actually done as a conversion.